packages/core/test/financial-templates/expiring-multiparty/PricelessPositionManager.js

```diff
@@ -1077,7 +1077,7 @@ contract("PricelessPositionManager", function(accounts) {
     const feesOwed = (
       await store.computeRegularFee(startTime.addn(1), startTime.addn(102), { rawValue: pfc.toString() })
     ).regularFee;
-    assert(Number(pfc.toString()) < Number(feesOwed.toString()));
+    assert.isTrue(Number(pfc.toString()) < Number(feesOwed.toString()));
```

Comment:
This came from looking through all financial-templates unit tests and removing all `asserts()` that are not asserting `didContractThrow`